lib/view_component/sub_component_wrapper.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+module ViewComponent
+  class SubComponentWrapper
+    attr_writer :_component_instance, :_content_block, :_content
```

Comment:
Why the underscore convention?

Reply:
The thought here was to try and limit the possible collisions that this class and the underlying `_component_instance` might have since this class is essentially a delegator.